src/js/BookReader.js

```diff
@@ -277,7 +283,11 @@ BookReader.prototype.initParams = function() {
     // params explicitly set in URL take precedence over all other methods
     var urlParams = this.paramsFromFragment(this.urlReadFragment());
     if (urlParams.mode) {
-      this.prevReadMode = urlParams.mode;
+      // If there were any parameters
+      if (Object.keys(urlParams).length > 0) {
```

Comment:
**Suggestion**: you could use `length` as a truthy value here and drop the comparison to zero.

Reply:
done